filecoin-proofs/src/bin/phase2.rs

```diff
@@ -295,18 +244,15 @@ fn blank_winning_post_poseidon_circuit(
         priority: false,
     };
 
-    let public_params = winning_post_public_params(&post_config).unwrap();
+    let public_params = winning_post_public_params::<DefaultOctTree>(&post_config).unwrap();
 
-    <FallbackPoStCompound<PoseidonHasher> as CompoundProof<
-        Bls12,
-        FallbackPoSt<PoseidonHasher>,
-        FallbackPoStCircuit<Bls12, PoseidonHasher>,
+    <FallbackPoStCompound<DefaultOctTree> as CompoundProof<
```

Comment:
Same question as above. How can we hard-code these types without reference to the sector size. It seems we need some variation of the `with_shape` macro.

Reply:
phase2 is not yet updated it seems

Reply:
Working on it

Reply:
@dignifiedquire Can you add a thumbs up here when you have checked give phase2.rs a once-over and feel confident that it has been updated accurately for the new changes?

Reply:
checked, fixed on thing, looks good now